_docs-sources/pipelines/how-it-works/index.md

```diff
@@ -1,7 +1,9 @@
 # How it works
 
-<!-- Placeholder for future content for Pipelines v2 -->
+TODO: Translate this into prose, or headers
 
-## Usage Data
-
-Gruntwork Pipelines captures usage data to better understand how our customers use our products. This information includes the duration of pipelines runs, the number of jobs you run, customer name, and application errors that occur during execution. If you would like to disable telemetry, please set the environment variable `PIPELINES_TELEMETRY_OPT_OUT=1` in your CI job.
+- User opens a Pull Request
+- Pipelines identifies all infra-changes being proposed
+- For each infra-change, Pipelines runs the relevant Pipelines Actions
+- The results of these actions show up as commnets and check runs in the GitHub Pull Request
+- [Future] Configure Pipelines to run different Pipelines Actions.
```

Comment:
We should aim for something basic, simple, and with a screenshot here so users can get it right away without having to install pipelines themselves.

Reply:
I generally don't like bulleted lists in docs, they present information in a way that makes you quickly scan it rather than absorb the context through a well written paragraph.

Reply:
The bulleted list here was intended as a placeholder, not the final product. But if there are stores in the backlog for this, that's probably a better place to do this work.

Reply:
ah i missed that, thanks!